src/applications/personalization/profile-2/sass/profile-info-table.scss

```diff
@@ -0,0 +1,65 @@
+.profile-info-table {
+  border-collapse: collapse;
+  border-spacing: 0;
+
+  th:first-child {
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+    border-bottom: none;
+    width: 100%;
+  }
+
+  tr:not(:last-child) {
+    td {
+      border-bottom: none;
+    }
+  }
+
+  tr:last-child {
+    td:last-child {
+      border-bottom-right-radius: 4px;
+      border-bottom-left-radius: 4px;
+    }
+  }
+
+  th,
+  td {
+    border: 1px solid $color-gray-lighter;
+    text-align: left;
+    display: flex;
+    padding: units(1.5) units(4);
+  }
+
+  tr {
+    display: flex;
+    flex-direction: row;
+    border: none;
+  }
+
+  td {
+    padding: 10px 30px;
+    width: 100%;
+  }
+
+  tr,
+  td:first-child {
+    border-right: none;
+    width: 75%;
+  }
+
+  tr,
+  td:last-child {
+    border-left: none;
+    width: 100%
+  }
```

Comment:
I'm curious if any or all of this should be applied to the design system's table, https://design.va.gov/components/tables.

Reply:
It might be worth having someone consider that later down the line. FWIW I need to change how this table works so it can work with the mobile layout it needs to render.

Reply:
Sorry, dumb question... doesn't adding a `display: flex` to a table cells & row totally break the semantic meaning of the table?

Reply:
Uh... probably? For a few reasons I realized that we need to move away from using a table and do this another way. I'm mostly done with rebuilding it and and will have a PR up soon.

Reply:
Uh... probably? For a few reasons I realized that we need to move away from using a table and do this another way. I'm mostly done with rebuilding it and and will have a PR up soon.